graylog2-server/src/main/java/org/graylog2/lookup/dto/DataAdapterDto.java

```diff
@@ -81,6 +101,15 @@ public static Builder builder() {
         @JsonProperty(FIELD_NAME)
         public abstract Builder name(String name);
 
+        @JsonProperty(FIELD_CUSTOM_ERROR_TTL_ENABLED)
+        public abstract Builder customErrorTTLEnabled(@Nullable Boolean enabled);
```

Comment:
Not having the option would store the default into each DataAdapter once it is edited by the user.
That means if we decide to change the default, some users might not benefit from our change, because we cannot tell, whether they wanted the default, or changed it on purpose.
It feels indeed like a lot of boilerplate, but since it's written already, I'd leave it as is :)

Reply:
Okay, fair enough. :+1: